src/commands.rs

```diff
@@ -535,1170 +480,1114 @@ pub enum KeySubCommand {
-            "bdk-cli",
-            "--network",
-            "bitcoin",
-            "external_reserves",
-            message,
-            psbt,
-            "6",
-            address,
-            address, // passing the address twice on purpose, to test passing of multiple addresses
-            "--server",
-            "ssl://electrum.blockstream.info:60002",
-        ];
-        let cli_opts = CliOpts::from_iter(&cli_args);
-
-        let (message, psbt, confirmations, addresses, electrum_opts) = match cli_opts.subcommand {
-            CliSubCommand::ExternalReserves {
-                message,
-                psbt,
-                confirmations,
-                addresses,
-                electrum_opts,
-            } => (message, psbt, confirmations, addresses, electrum_opts),
-            _ => panic!("unexpected subcommand"),
-        };
-        let result = handle_ext_reserves_subcommand(
-            Network::Bitcoin,
-            message,
-            psbt,
-            confirmations,
-            addresses,
-            electrum_opts,
-        )
-        .unwrap();
-        let spendable = result
-            .as_object()
-            .unwrap()
-            .get("spendable")
-            .unwrap()
-            .as_u64()
-            .unwrap();
-        assert!(spendable > 0);
-    }
-
-    #[cfg(feature = "repl")]
-    #[test]
-    fn test_regex_double_quotes() {
-        let split_regex = Regex::new(crate::REPL_LINE_SPLIT_REGEX).unwrap();
-        let line = r#"restore -m "word1 word2 word3" -p 'test! 123 -test' "#;
-        let split_line: Vec<&str> = split_regex
-            .captures_iter(&line)
-            .map(|c| {
-                c.get(1)
-                    .or_else(|| c.get(2))
-                    .or_else(|| c.get(3))
-                    .unwrap()
-                    .as_str()
-            })
-            .collect();
-        assert_eq!(
-            vec!(
-                "restore",
-                "-m",
-                "word1 word2 word3",
-                "-p",
-                "test! 123 -test"
-            ),
-            split_line
-        );
-    }
-
-    #[cfg(feature = "repl")]
-    #[test]
-    fn test_regex_single_quotes() {
-        let split_regex = Regex::new(crate::REPL_LINE_SPLIT_REGEX).unwrap();
-        let line = r#"restore -m 'word1 word2 word3' -p "test *123 -test" "#;
-        let split_line: Vec<&str> = split_regex
-            .captures_iter(&line)
-            .map(|c| {
-                c.get(1)
-                    .or_else(|| c.get(2))
-                    .or_else(|| c.get(3))
-                    .unwrap()
-                    .as_str()
-            })
-            .collect();
-        assert_eq!(
-            vec!(
-                "restore",
-                "-m",
-                "word1 word2 word3",
-                "-p",
-                "test *123 -test"
-            ),
-            split_line
-        );
-    }
-}
+// #[cfg(test)]
+// mod test {
```

Comment:
Will all of the tests here ultimately need to be updated in a follow-up PR?

Reply:
However, if you notice any outputs that seem incorrect or if you want specific features or sections tested, please indicate them here.